mappings/net/minecraft/server/world/SimpleTickScheduler.mapping

```diff
@@ -1,6 +1,25 @@
 CLASS net/minecraft/class_4296 net/minecraft/server/world/SimpleTickScheduler
 	FIELD field_19275 scheduledTicks Ljava/util/List;
 	FIELD field_19276 identifierProvider Ljava/util/function/Function;
+	METHOD <init> (Ljava/util/function/Function;Ljava/util/List;)V
+		ARG 1 identifierProvider
+		ARG 2 scheduledTicks
+	METHOD <init> (Ljava/util/function/Function;Ljava/util/List;J)V
+		ARG 1 identifierProvider
+		ARG 2 scheduledTicks
+		ARG 3 startTime
 	METHOD method_20463 toNbt ()Lnet/minecraft/class_2499;
 	METHOD method_20512 fromNbt (Lnet/minecraft/class_2499;Ljava/util/function/Function;Ljava/util/function/Function;)Lnet/minecraft/class_4296;
 		ARG 0 ticks
+	METHOD method_26368 scheduleTo (Lnet/minecraft/class_1951;)V
+		ARG 1 scheduler
+	CLASS class_4989 Tick
+		FIELD field_23257 pos Lnet/minecraft/class_2338;
+		FIELD field_23258 delay I
+		FIELD field_23259 priority Lnet/minecraft/class_1953;
+		FIELD field_23260 object Ljava/lang/Object;
+		METHOD <init> (Ljava/lang/Object;Lnet/minecraft/class_2338;ILnet/minecraft/class_1953;)V
+			ARG 1 object
```

Comment:
isn't `o` the preferred name for object parameters? at least saw it on a recent pr

Reply:
That o is for like universal object, this is more like a generic value in the tick scheduler